packages/edge/README.md

```diff
@@ -157,7 +166,7 @@ sendEvent(experienceEvent: ExperienceEvent): Promise<Array<EdgeEventHandle>>
 ```typescript
 var xdmData  = {"eventType" : "SampleXDMEvent"};
 var data  = {"free": "form", "data": "example"};
-let experienceEvent = new ExperienceEvent(xdmData, data);
+let experienceEvent = new ExperienceEvent({xdmData: xdmData , data: data});
```

Comment:
when the key and value are the same, you can just specify it once :)

Reply:
I changed them to different names, so I can keep the keys.